packages/report-server/src/reportBuilder/output/functions/matrix/types.ts

```diff
@@ -0,0 +1,14 @@
+import { Row } from '../../../types';
+
+export type MatrixParams = {
+  columns: { include: string[] | '*'; exclude: string[] };
```

Comment:
```suggestion
  columns: { includeFields: string[] | '*'; excludeFields: string[] };
```
After seeing your code in `matrixBuilder.ts` I feel like you got the terms just right there 👍 
Optional ofcourse :)